build/teamcity/cockroach/nightlies/pebble_nightly_metamorphic_crossversion.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+#
+# This script is run by the Pebble Nightly Crossversion Metamorphic - TeamCity
+# build configuration.
+
+set -euo pipefail
+
+dir="$(dirname $(dirname $(dirname $(dirname "${0}"))))"
+
+source "$dir/teamcity-support.sh"  # For $root
+source "$dir/teamcity-bazel-support.sh"  # For run_bazel
+
+mkdir -p bin
+chmod o+rwx bin
+mkdir -p $root/artifacts
+
+bazel build //pkg/cmd/bazci --config=ci
+
+VERSIONS=""
+LAST_SHA=""
+for branch in "$@"
+do
+    tc_start_block "Compile Pebble $branch metamorphic test binary"
+    SHA=$($dir/teamcity/cockroach/nightlies/pebble_nightly_build_test_binary.sh "$branch" "bin")
```

Comment:
The build will also output a lot of stuff to `stdout`. You can address this by piping to `tail` like:

    SHA=$("$dir/teamcity/cockroach/nightlies/pebble_nightly_build_test_binary.sh" "$branch" "bin" | tail -n1)